modules/standard/Math.chpl

```diff
@@ -1195,7 +1195,12 @@ module Math {
     return a;
   }
 
-
+  /* Returns true if `x` and `y` are approximately equal, else returns false. */
+  inline proc isclose(x, y, rtol = 1e-5, atol = 1e-8): bool {
+    if boundsChecking && (rtol <= 0 || atol < 0) then
```

Comment:
I'd much rather have these checks for two arguments separated. This way the error message can be more concise.

In that case, if both of the arguments are passed illegal values it may be OK to printout the error for only one of them first just to make the implementation easy. It'd be great if we check both and print out both messages, but I don't feel it's a must have.

Reply:
I just assumed that since it was such a simple error to debug, it'd be okay to club the message in one and print it out.

I'll separate the messages and make it more concise.

Reply:
I agree with that. I just prefer a shorter error message. Another alternative I thought of was only say something like "Arguments to isclose is wrong" but then it is not as verbose as I'd prefer. (I think it'd be nice for the error message to mention the name of the argument)